orchestra/views.py

```diff
@@ -336,7 +336,7 @@ class TimeEntryList(generics.ListCreateAPIView):
     permission_classes = (permissions.IsAuthenticated,)
     serializer_class = TimeEntrySerializer
     filter_backends = (filters.DjangoFilterBackend,)
-    filter_class = TimeEntryFilter
+    filterset_class = TimeEntryFilter
```

Comment:
was this just always broken?:) how'd you find it?

Reply:
That's because I updated `django-filter`. Since they added django2 support only in v2.0